client/reader/following/intro.jsx

```diff
@@ -13,44 +13,65 @@ import { bindActionCreators } from 'redux';
 import QueryPreferences from 'components/data/query-preferences';
 import { savePreference } from 'state/preferences/actions';
 import { getPreference } from 'state/preferences/selectors';
+import { recordTrack } from 'reader/stats';
 
-const FollowingIntro = ( props ) => {
+class FollowingIntro extends React.Component {
 
-	if ( ! props.isNewReader ) {
-		return null;
+	componentDidMount() {
+		this.recordRenderTrack();
 	}
 
-	return (
-		<header className="following__intro">
-			<QueryPreferences />
-			<div className="following__intro-header">
-				<div className="following__intro-copy">
-						{ props.translate(
-							'{{strong}}Welcome!{{/strong}} Reader is a custom magazine. ' +
-							'{{link}}Follow your favorite sites{{/link}} and their latest ' +
-							'posts will appear here. {{span}}Read, like, and comment in a ' +
-							'distraction-free environment.{{/span}}',
-							{
-								components: {
-									link: <a href="/following/edit" />,
-									strong: <strong />,
-									span: <span className="following__intro-copy-hidden" />
+	componentWillReceiveProps( nextProps ) {
+		if ( this.props.isNewReader !== nextProps.isNewReader ) {
+			this.recordRenderTrack( nextProps );
+		}
+	}
+
+	recordRenderTrack = ( props = this.props ) => {
+		if ( props.isNewReader === true ) {
+			recordTrack( 'calypso_reader_following_intro_render' );
+		}
+	}
+
+	render() {
+		const { isNewReader, translate, dismiss } = this.props;
+
+		if ( ! isNewReader ) {
+			return null;
+		}
+
+		return (
+			<header className="following__intro">
+				<QueryPreferences />
+				<div className="following__intro-header">
+					<div className="following__intro-copy">
+							{ translate(
+								'{{strong}}Welcome!{{/strong}} Reader is a custom magazine. ' +
+								'{{link}}Follow your favorite sites{{/link}} and their latest ' +
+								'posts will appear here. {{span}}Read, like, and comment in a ' +
+								'distraction-free environment.{{/span}}',
+								{
+									components: {
+										link: <a href="/following/edit" />,
```

Comment:
maybe this should respect the feature flag with https://github.com/Automattic/wp-calypso/pull/13779 landing?